### ISSUE TITLE
We should get these deployed on Cacti, and also, 

### ISSUE THREAD

**Issue by 6100m** [WiiLink24/WC24Cron#1]
I also do want to finish that OSM port to News Channel, on top of even adding OSM to Forecast Channel if we can.

**Comment by 6100m** [comment]
Git-Pull-Cron, and add-cron, and room-server are the three dependcies, and room-server needs it from my gloom-utils branch too.

**Comment by 6100m** [comment]
@spotlightishere Hey spotlight, could you maybe hold off on deploying it for about 5-10 mins? Some of the stuff is broken, so Imma fix it first :) Ill post a reply when I am ready :)

**Comment by spotlightishere** [comment]
sure thing :)

**Comment by 6100m** [comment]
@spotlightishere Alrighty spotlight, its ready 😄 
To configure it, do
su root
mkdir /var/share/var/cron/main/
cd /var/share/var/cron/main/
git clone https://github.com/WiiLink24/WiiCron24.git .
sudo yum install nohup
mkdir installlog
sudo yum install npm
npm install git-pull-cron
npm install forever
sudo ./runme.sh
cp startbot.sh installlog/startbot.sh
cd installlog
nohup ./startbot.sh 0<&- &> main.log &

**Comment by spotlightishere** [comment]
First could you please update package.json to have those two dependencies so we can keep them updated?

**Comment by 6100m** [comment]
Yeah sure thing, one min :)

**Comment by 6100m** [comment]
Alrighty spotlight, I got that updated :) One of the cool features about this engine is that it updates code dynamically all by itself :)

**Comment by spotlightishere** [comment]
I'm not quite sure what's up with `runme.sh`, when you get a chance could you please investigate?
```
Running...
Do you wish to install this program?y
mkdir: cannot create directory ‘/var/share/var/cron/main/’: File exists
sudo: ./auto.sh: command not found
Done.
```

**Comment by 6100m** [comment]
I think it already exists for some reason, huh. As for the auto.sh problem, I know whats happening :) One min

**Comment by 6100m** [comment]
@spotlightishere 
su root
rm -rf /var/share/var/cron/main/
mkdir /var/share/var/cron/main/
cd /var/share/var/cron/main/
git clone https://github.com/WiiLink24/WiiCron24.git .
sudo yum install nohup
mkdir installlog
sudo yum install npm
npm install git-pull-cron
npm install forever
sudo ./runme.sh
cp startbot.sh installlog/startbot.sh
cd installlog

To run the final task in production mode, do
nohup ./startbot.sh -u spotlight -a spotlight -g gloom 0<&- &> main.log &
cd ../
To run the final task in debug mode, do
nohup ./startbot.sh -u gloom -a spotlight -g gloom 0<&- &> secn.log &
cd ../

I named the args for that script after both of our usernames :)

**Comment by 6100m** [comment]
And if it gives the directory error, don't worry, that is just a failsafe :)

**Comment by spotlightishere** [comment]
It's still not working, I'm sorry.. could you please create a VM or something locally and debug this? I unfortunately can't get it to work due to a few errors, such as a syntax error within package.json due to missing commas. ;p

**Comment by 6100m** [comment]
Yeah, will do when I get the chance :) Could we maybe get the galvatron engine deployed instead maybe then? I do wanna try to get one of the engines deployed so after my schoolwork, it'll be close to done :)

**Comment by 6100m** [comment]
@zurgeg Hey zurgeg, could you go in and fix up the scripts maybe? I might try with this in a few days, but I was wondering if you could maybe fix the script errors?

**Comment by 6100m** [comment]
@spotlightishere I came up with a way to do this manually. Its a bit of a stretch, but it should be fine. 😄 
su root
rm -rf /var/share/var/cron/main/
rm -rf /var/share/var/bot
mkdir /var/share/var/cron/main/
cd /var/share/var/cron/main/
mkdir main
mkdir secn
git clone https://github.com/WiiLink24/WiiCron24.git main/
sudo yum install nohup
sudo yum install npm
npm install git-pull-cron
npm install forever
git clone https://github.com/WiiLink24/room-server.git secn/ --single-branch --branch gloom-utils
cd secn
crontab gloom/srv/env.cron
cd ../
cd main
cd all
crontab jobs.cron
cd ../
cd Engine/assets/bin/
sh init.sh
sh path.sh
cd ../
forever start update.js

That should maybe work? If not, yeah, I'm going to debug this in a VM :) I just wanna see if it runs manually good, to see if it's only the scripts that are broken :)

**Comment by 6100m** [comment]
I'll be debugging too though, but yeah, hopefully we can figure out if it's the scripts or the commands themselves. :)
